tests/setupAfterEnv.ts

```diff
@@ -47,7 +48,9 @@ expect.extend({
    */
   async toResolve(promise: Promise<any>) {
     if (this.isNot) {
-      throw new Error('Using `.not.toResolve(...)` is not supported.');
+      throw new Error(
+        'Using `.not.toResolve()` is not supported. Use .toReject() instead.',
```

Comment:
Was this meant to be `toBeRejected`?

`toReject` is more in-line with `toResolve` though.

Reply:
I updated this error message since I removed `toBeRejected`.